src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookConsumer.java

```diff
@@ -56,41 +56,68 @@ void process(AbstractWebhookEvent e) {
 
     private void process(RefsChangedWebhookEvent event) {
         BitbucketRepository repository = event.getRepository();
-        LOGGER.fine(format("Received refs changed event from repo: %s/%s  ", repository.getProject().getKey(), repository.getSlug()));
-        if (!isEligibleRefs(event)) {
-            return;
-        }
-        RefChangedDetails refChangedDetails = new RefChangedDetails(event);
-        triggerJob(event, refChangedDetails);
-    }
-
-    private void process(MirrorSynchronizedWebhookEvent event) {
-        BitbucketRepository repository = event.getRepository();
-        LOGGER.fine(format("Received Mirror Synchronized changed event from repo: %s/%s  ", repository.getProject().getKey(), repository.getSlug()));
-        if (!isEligibleRefs(event)) {
-            return;
+        String webhookEventType = "refs changed";
```

Comment:
nit, but I think we should either change this to `Refs Changed` or change the mirror one to `mirror synchronized` and avoid mixing of casing.

Reply:
I'm happy to do that. I did this to ensure the logging was identical to the previous version just in case that matters, but if you don't think it's a big deal I'd rather make the casing consistent.